engines/metaengine.h

```
@@ -313,200 +343,226 @@ class MetaEngine : public PluginObject {
+		* unavailable. In that case Save/Load dialog for the engine's
 		* games is locked during cloud saves sync.
 		*/
 		kSimpleSavesNames,
 
 		/**
-		 * Uses default implementation of save header and thumbnail
+		 * Use the default implementation of save header and thumbnail
 		 * appended to the save.
+		 *
 		 * This flag requires the following flags to be set:
-		 *   kSavesSupportMetaInfo, kSavesSupportThumbnail, kSavesSupportCreationDate,
-		 *   kSavesSupportPlayTime
+		 * - kSavesSupportMetaInfo
+		 * - kSavesSupportThumbnail
+		 * - kSavesSupportCreationDate
+		 * - kSavesSupportPlayTime
 		 */
 		kSavesUseExtendedFormat
 	};
 
-	//@}
-
 	/**
 	 * Return a list of achievement descriptions for the specified target.
 	 *
 	 * The default implementation returns an empty list.
 	 *
-	 * @param target    name of a config manager target
-	 * @return          a list of achievement descriptions for an engine plugin
-	 *                  and target
+	 * @param target  Name of a config manager target.
+	 *
+	 * @return A list of achievement descriptions for an engine plugin and target.                
 	 */
 	virtual const Common::AchievementsInfo getAchievementsInfo(const Common::String &target) const {
 		return Common::AchievementsInfo();
 	}
 
 	/**
 	 * Determine whether the engine supports the specified MetaEngine feature.
-	 * Used by e.g. the launcher to determine whether to enable the "Load" button.
+	 *
+	 * Used by e.g. the launcher to determine whether to enable the Load button.
 	 */
 	virtual bool hasFeature(MetaEngineFeature f) const;
 
+	/**
+	 * @todo Append extended save (?).
+	 */
 	static void appendExtendedSave(Common::OutSaveFile *saveFile, uint32 playtime, Common::String desc, bool isAutosave);
+	/**
+	 * @todo Parse savegame header to retrieve its description.
+	 */
 	static void parseSavegameHeader(ExtendedSavegameHeader *header, SaveStateDescriptor *desc);
+	/**
+	 * @todo Create a dummy savegame header.
```

Populate the given extended savegame header with dummy values. This is used when failing to read the header from a savegame file.